test/executable/executableTests.ts

```diff
@@ -79,6 +79,17 @@ describe("Executable", function() {
                 done();
             });
         });
+
+        it("exits with code 1 if config file is invalid", (done) => {
+            execCli(["-c", "test/config/tslint-invalid.json", "src/tslint.ts"], (err, stdout, stderr) => {
+                assert.isNotNull(err, "process should exit with error");
+                assert.strictEqual(err.code, 1, "error code should be 1");
+
+                assert.include(stderr, "Invalid option for configuration", "stderr should contain notification about invalid json");
```

Comment:
this test isn't exercising the code in this PR. `Invalid option for configuration` is written when the config file is missing. I was expecting `Failed to load <path>: <error message>`